ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/PulseZoneSettings.java

```diff
@@ -1,88 +1,196 @@
 package com.dsi.ant.antplus.pluginsampler.pulsezones;
 
-import android.os.Parcel;
-import android.os.Parcelable;
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.dsi.ant.antplus.pluginsampler.R;
 
 import java.util.Arrays;
 
 /**
  * Created by ksenia on 01.01.19.
  */
 
-public class PulseZoneSettings implements Parcelable {
+public class PulseZoneSettings {
+    private static final String PARAM_APP_PREFERENCES = "prefID";
     private int gender;
```

Comment:
Let's rename globally the gender/sex to something like "selectedGenderRadio" 
The same for the "zone", to be clear that these fields represent the radio button selection